### PR TITLE
Carla Coupling Adaptions

### PR DESCRIPTION
**Pull request by realmaxneu**
## Type of this PR 

- [ ] Bug fix
- [x] Enhancement

## Description

This PR contains a few necessary adaptions for the Carla coupling.
 
## Affected parts of the online documentation

Are there any parts in the online documentation which are affected by this PR?  
-> No.

## Definition of Done

- [x] You have read CONTRIBUTING.md carefully.
- [x] You have signed the [Contributor License Agreement](http://www.eclipse.org/legal/CLA.php).
- [x] All commits are signed-off (`-s`) with your mail address of your Eclipse Account.
- [x] `origin/main` has been merged into your Fork.
- [ ] New functionality is covered by unit tests or integration tests. Code coverage must not decrease.
- [ ] There are no new SpotBugs warnings. 
- [x] Coding guidelines have been observed (see CONTRIBUTING.md).
- [ ] All tests pass.

## Special notes to reviewer

It should be examined if the change from `VehicleSetMoveToXY.Mode.KEEP_ROUTE` to `VehicleSetMoveToXY.Mode.EXACT_POSITION` in fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/AbstractSumoAmbassador.java leads to any misbehaviour when other traffic simulators than Carla are coupled together with SUMO to Mosaic.

### CODE REVIEW

File: lib/mosaic-interactions/src/main/java/org/eclipse/mosaic/interactions/vehicle/VehicleSensorActivation.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+package org.eclipse.mosaic.interactions.vehicle;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.eclipse.mosaic.rti.api.Interaction;
+
+import static org.apache.commons.lang3.builder.ToStringStyle.SHORT_PREFIX_STYLE;
+
+public final class VehicleSensorActivation extends Interaction {
```
Comment (kschrab) [inline lib/mosaic-interactions/src/main/java/org/eclipse/mosaic/interactions/vehicle/VehicleSensorActivation.java:25]: Could we merge this with `VehicleDistanceSensorActivation`?

Comment (kschrab) [reply lib/mosaic-interactions/src/main/java/org/eclipse/mosaic/interactions/vehicle/VehicleSensorActivation.java:25]: We also should extend `VehicleOperatingSystem` to create/trigger this interaction.

Comment (felixlutz) [reply lib/mosaic-interactions/src/main/java/org/eclipse/mosaic/interactions/vehicle/VehicleSensorActivation.java:25]: Done in PR #146

**Comment by kschrab** [comment]
Done with #146

**Pull request closed without merging**